src/drag-and-drop/components/draggable-item-wrapper.test.tsx

```diff
@@ -9,7 +9,7 @@ describe("DraggableItemWrapper", () => {
   it("renders DraggableItem", () => {
     const wrapper = mount(
       <DndProvider backend={TouchBackend} options={{enableMouseEvents: true}} >
-        <DraggableItemWrapper item={{ id: "1", imageUrl: "https://image/1" }} position={{ left: 1, top: 1}} draggable={true} />
+        <DraggableItemWrapper item={{ id: "1", imageUrl: "https://image/1", imageWidth: 25, imageHeight: 25}} position={{ left: 1, top: 1}} draggable={true} />
```

Comment:
You might break the props onto multiple lines for readability and then check spacing on the objects.
```
<DraggableItemWrapper 
  item={{id: "1", imageUrl: "https://image/1", imageWidth: 25, imageHeight: 25}} 
  position={{left: 1, top: 1}} 
  draggable={true} 
/>